content/docs/start/experiments.md

```diff
@@ -290,3 +290,7 @@ $ dvc exp show -n 2 --no-timestamp \
 > `dvc exp gc` only removes references to the experiments; not the cached
 > objects associated with them. To clean up the <abbr>cache</abbr>, use
 > `dvc gc`.
+
+Checkout our video on Experiments.
+
+https://youtu.be/N11KcrtbVDY
```

Comment:
These typically go near the top in other GS pages. No intro sentence needed I think.

Reply:
Ok.  I'll take a look.  I accidentally put the checkpoints video again for the experiments one so I just had to change anyway.